src/pl-fli.c

```diff
@@ -4391,11 +4391,11 @@ PL_raise_exception(term_t exception)
 
   LD->exception.processing = TRUE;
   if ( !PL_same_term(exception, exception_bin) ) /* re-throwing */
-  { except_class co = classify_exception(exception_bin);
-    except_class cn = classify_exception(exception);
+  { except_class class_old = classify_exception(exception_bin);
```

Comment:
I had to re-read this code a couple of times to understand it (this might be more a reflection on me than on the code). At minimum, having two variables with a single letter different can lead to typos errors. How about `c_now` and `c_old`, if my suggested names are too long?

Reply:
I don't really care about the length of the name, although I prefer short names when they are used very locally and it is clear enough from the context what they mean.    I mostly dislike patches for such small changes as it merely makes it harder to deal with the history of the code.

Reply:
As long as my documentation is correct, there's no need to apply my patch. (For the reason you gave)

Shall I delete both this PR (and the other one that didn't have the biblio commit) and make a new PR?